src/style.css

```diff
@@ -0,0 +1,121 @@
+:root {
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --color-of-button: #00acdc;
+}
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
Don't reset margin using * selector. Better specify the exact tags you want to reset.
Same with the font. Just set it for the whole page and inherit where needed